Makefile

```diff
@@ -0,0 +1,23 @@
+
+PHONY: test verify go-bootstrap
+
+build: 
+	godep go build .
+
+# test will run the python tests using the cli 
+test: build 
+	trial -j2 dvol_python \
```

Comment:
We had to remove the `-j2` flag from the `trial` command in a different branch because it running tests in parallel doesn't interact well with hypothesis so please remove it here.